src/console/Console.cxxtest

```diff
@@ -22,7 +22,7 @@ TEST(ConsoleTest, testHelp)
                               "             has an effect on socket based logins sessions\n"
                               "> ", 154));
 
-    EXPECT_EQ(::write(s, "?\n", 3), 3);
+    EXPECT_EQ(::write(s, "?\n", 5), 5);
```

Comment:
This change does not seem right. There are only three characters in that string (question mark, newline, zero). We should not write 5 bytes here.